internal/service/project/resource_project.go

```diff
@@ -639,16 +654,48 @@ func GetProjectPropsFromAPI(ctx context.Context, projectsAPI admin.ProjectsApi,
 	if err != nil {
 		return nil, fmt.Errorf("error getting project's IP addresses (%s): %v", projectID, err.Error())
 	}
+	IsSlowOperationThresholdingEnabled, err := ReadIsSlowMsThresholdingEnabled(ctx, performanceAdvisorAPI, projectID)
```

Comment:
```suggestion
	isSlowOperationThresholdingEnabled, err := ReadIsSlowMsThresholdingEnabled(ctx, performanceAdvisorAPI, projectID)
```

Reply:
Thanks. Updated in 53981b328430c611ad1b21fd1d31c2ea9678eff1